README.md

```diff
@@ -251,3 +251,24 @@ it needs to run as a plugin to Anki desktop to generate notes to a local
 collection. It does not seem to allow editing/adding to a local collection
 outside of Anki desktop.
 
+## Contributing
+
+### Setup
+
+First fork the repository, then clone your fork and install the package
+dependencies:
+
+```console
+$ git clone git@github.com:<username>/apy.git
+$ cd apy/
+$ mkvirtualenv anki -p python3.8
+$ echo "anki" > .venv  # if you use a loader like zsh-autoswitch-virtualenv
+$ ./setup.py install
+$ pip install -r requirements-dev.txt
```

Comment:
Does this now also install anki itself?

Reply:
Right. I was surprised that anki wasn't listed in the setup install_requires, so I added it here for the sake of having all related packages in the same virtual environment.

Reply:
Huh. So, do I understand correctly that these steps would actually work with regard to starting fresh and getting a virtual environment in which `apy` works? (I would not have thought it could be so simple.)

Reply:
well... yes.

Note that the dev setup needs more improvements, like defining a custom location for the anki db (I believe it always default to `~/.local/share/Anki2`, and you wouldn't want to write unittests that modify your real anki db ^^).

Reply:
That's true. Note, though, that I've added a test db that is specified in the tests, and the tests should never touch your personal anki db. However, if you do manual testing, then it is useful to secify the `-b` option and use a custom, temporary database.